pennylane_lightning/src/simulators/lightning_qubit/gates/tests/Test_GateImplementations_CompareKernels.cpp

```diff
@@ -31,6 +43,7 @@ namespace {
 using namespace Pennylane::LightningQubit;
 using namespace Pennylane::LightningQubit::Util;
 using namespace Pennylane::LightningQubit::Gates;
```

Comment:
I may ask for a brief summary (again) if I didn't do this earlier, but the difference between `Pennylane::LightningQubit::Gates` and `Pennylane::Gates` is something I have forgotten.

Reply:
`Pennylane::Gates` => device-agnostic namespace.
`Pennylane::LightningQubit::Anything` => device-specific

Reply:
The first has LQubit-specific definitions, like those touching to AVX kernels. The latter has generic definitions, like functions that generate various gates in vector form.